### PR TITLE
MDP-192: Thumbnail component

### PR DESCRIPTION
**Pull request by slavomircesla**
https://velocity.globallogic.com/jira/browse/MDP-192

### CODE REVIEW

File: src/Router.tsx [hunk 1]
```diff
@@ -3,15 +3,13 @@ import React from "react";
 import Home from "./components/Home";
 import Example from "./components/Example";
 import Counter from "./components/Counter";
-import Testing from "./components/Testing";
```
Comment (pggl) [inline src/Router.tsx]: revert this changes

File: src/components/thumbnailTherapy/TherapyButton/TherapyButton.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const TherapyButton = () => {
+  return (
+    <button className=" rounded-lg bg-transparent w-full h-10 font-bold text-black hover:text-white hover:bg-black text-black ">
+      Zistiť viac
```
Comment (pggl) [inline src/components/thumbnailTherapy/TherapyButton/TherapyButton.tsx]: we need props for this

File: src/components/thumbnailTherapy/TherapyDescription/TherapyDescription.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const ThumbnailDescr =
+  "Kreatívne a uvoľnujúce aktivity, aby ste si oddýchli, spracovali emócie […]";
+
+const TherapyDescription = () => {
+  return (
+    <p className=" text-sm leading-normal font-lato font-normal mb-4 ">
+      {ThumbnailDescr}
```
Comment (pggl) [inline src/components/thumbnailTherapy/TherapyDescription/TherapyDescription.tsx]: change this by props

File: src/components/thumbnailTherapy/TherapyImage/TherapyImage.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import picture from "./homer.svg";
+
+const TherapyImage = () => {
+  return <img className="mb-2" src={picture} />;
```
Comment (pggl) [inline src/components/thumbnailTherapy/TherapyImage/TherapyImage.tsx]: change this by props

File: src/main.css [hunk 5]
```diff
@@ -0,0 +1,137 @@
+html,
```
Comment (pggl) [inline src/main.css]: remove this

File: src/stories/index.tsx [hunk 6]
```diff
@@ -1,7 +1,18 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
 import Example from "../components/Example";
-import "../style/index.css";
-import "../style/fonts.css";
```
Comment (pggl) [inline src/stories/index.tsx:9]: revert this changes we need this

File: tailwind.js [hunk 7]
```diff
@@ -119,9 +119,9 @@ module.exports = {
   */
 
   fonts: {
-    "gilbert": ["Gilbert", "serif"],
+    gilbert: ["Gilbert", "serif"],
```
Comment (pggl) [inline tailwind.js]: revert all this changes

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/thumbnailTherapy/Therapy.tsx [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+import TherapyButton from "./TherapyButton/TherapyButton";
+import TherapyDescription from "./TherapyDescription/TherapyDescription";
+import TherapyImage from "./TherapyImage/TherapyImage";
+import TherapyName from "./TherapyName/TherapyName";
+import icon from "./TherapyImage/homer.svg";
+
+const Therapy = () => {
+  return (
+    <div className="container ml-6 ">
+      <div className="p-2 w-1/5 h-1/3 m-2 shadow mr-2 mb-2 block ">
```
Comment (garySiladi) [inline src/components/thumbnailTherapy/Therapy.tsx]: delete space after `block`

File: src/components/thumbnailTherapy/TherapyButton/TherapyButton.tsx [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import React, { FC } from "react";
+
+interface IButton {
+  text: string;
+}
+
+const TherapyButton: FC<IButton> = ({ text }) => {
+  return (
+    <button className=" rounded-lg bg-transparent w-full h-10 font-black font-lato text-black hover:text-white hover:bg-black text-black font-black">
```
Comment (garySiladi) [inline src/components/thumbnailTherapy/TherapyButton/TherapyButton.tsx]: delete space in the beginning of the className

File: src/components/thumbnailTherapy/TherapyDescription/TherapyDescription.tsx [hunk 10]
```diff
@@ -0,0 +1,13 @@
+import React, { FC } from "react";
+
+export interface IDescribe {
+  text: string;
+}
+
+const TherapyDescription: FC<IDescribe> = ({ text }) => {
+  return (
+    <p className=" text-sm leading-normal font-lato font-normal mb-4">{text}</p>
```
Comment (garySiladi) [inline src/components/thumbnailTherapy/TherapyDescription/TherapyDescription.tsx]: delete space in the beginning of the className

File: src/components/thumbnailTherapy/TherapyName/TherapyName.tsx [hunk 11]
```diff
@@ -0,0 +1,15 @@
+import React, { FC } from "react";
+
+interface IName {
+  text: string;
+}
+
+const TherapyName: FC<IName> = ({ text }) => {
+  return (
+    <p className="font-Gilbert font-bold roman text-xl text-black leading-1.33 ">
```
Comment (garySiladi) [inline src/components/thumbnailTherapy/TherapyName/TherapyName.tsx]: delete space in the end of the className

**Review by garySiladi — CHANGES_REQUESTED** [review]

File: src/components/thumbnailTherapy/Therapy.tsx [hunk 12]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import TherapyButton from "./TherapyButton/TherapyButton";
+import TherapyDescription from "./TherapyDescription/TherapyDescription";
+import TherapyImage from "./TherapyImage/TherapyImage";
+import TherapyName from "./TherapyName/TherapyName";
+import icon from "./TherapyImage/homer.svg";
+
+const Therapy = () => {
+  return (
+    <div className="container ml-6 ">
```
Comment (garySiladi) [inline src/components/thumbnailTherapy/Therapy.tsx]: remove space

**Review by garySiladi — DISMISSED** [review]
nice work

File: src/components/thumbnailTherapy/Therapy.tsx [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+        <TherapyImage icon={icon} />
+        <TherapyName text="MAMA CHAT" />
+        <TherapyDescription text="Kreatívne a uvoľnujúce aktivity, aby ste si oddýchli, spracovali emócie […]" />
+        <TherapyButton text="Zistiť viac" />
+      </div>
+    </div>
+  );
+};
+
+export default Therapy;
```
Comment (pggl) [inline src/components/thumbnailTherapy/Therapy.tsx]: you have to run npm install and fix all issues, default export is forbidden

Comment (pggl) [reply src/components/thumbnailTherapy/Therapy.tsx]: and there is bad name for component

File: src/components/thumbnailTherapy/TherapyButton/TherapyButton.tsx [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+
+const TherapyButton: FC<IButton> = ({ text }) => {
+  return (
+    <button className="rounded-lg bg-transparent w-full h-10 font-black font-lato text-black hover:text-white hover:bg-black text-black font-black">
+      {text}
+    </button>
+  );
+};
+
+export default TherapyButton;
```
Comment (pggl) [inline src/components/thumbnailTherapy/TherapyButton/TherapyButton.tsx]: we don't need this subfolders, we need only thumbnailTherapy folder, or just name this folder thumbnail, so under this folder it is ok to just name this component Button. It will looks like src/component/thumbnail/Button.tsx, also other compents needs refactor

File: src/components/thumbnailTherapy/TherapyButton/TherapyButton.tsx [hunk 15]
```diff
@@ -0,0 +1,15 @@
+import React, { FC } from "react";
+
+interface IButton {
```
Comment (pggl) [inline src/components/thumbnailTherapy/TherapyButton/TherapyButton.tsx]: rename IButton to IProps, refactor another componets

File: src/components/thumbnailTherapy/TherapyName/TherapyName.tsx [hunk 16]
```diff
@@ -0,0 +1,15 @@
+import React, { FC } from "react";
+
+interface IName {
+  text: string;
+}
+
+const TherapyName: FC<IName> = ({ text }) => {
+  return (
+    <p className="font-Gilbert font-bold roman text-xl text-black leading-1.33">
```
Comment (pggl) [inline src/components/thumbnailTherapy/TherapyName/TherapyName.tsx]: do we need "roman" in className ?

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/Thumbnail/Thumbnail.tsx [hunk 17]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Button } from "./Button";
+import { Description } from "./Description";
+import { Image } from "./Image";
+import { Name } from "./Name";
+import icon from "./homer.svg";
+
+export const Thumbnail = () => {
+  return (
```
Comment (garySiladi) [inline src/components/Thumbnail/Thumbnail.tsx]: remove `return` and simply let the const return the render as
```
export const Thumbnail = () => (
  <div className="container ml-6">
  ....
```

File: src/components/Thumbnail/Name.tsx [hunk 18]
```diff
@@ -0,0 +1,13 @@
+import React, { FC } from "react";
+
+interface IProps {
+  text: string;
+}
+
+export const Name: FC<IProps> = ({ text }) => {
+  return (
```
Comment (garySiladi) [inline src/components/Thumbnail/Name.tsx]: remove `return` and simply let the const return the render as
```
export const Name: FC<IProps> = ({ text }) => (
  <p className="font-Gilbert font-bold text-xl text-black leading-1.33">
  ....
```

File: src/components/Thumbnail/Image.tsx [hunk 19]
```diff
@@ -0,0 +1,8 @@
+import React, { FC } from "react";
+
+interface IProps {
+  icon: string;
+}
+export const Image: FC<IProps> = ({ icon }) => {
+  return <img className="mb-2" alt="therapy-Image" src={icon} />;
```
Comment (garySiladi) [inline src/components/Thumbnail/Image.tsx]: remove `return` and simply let the const return the render as
```
export const Image: FC<IProps> = ({ icon }) => (
  <img className="mb-2" alt="therapy-Image" src={icon} />
  ....
```

File: src/components/Thumbnail/Description.tsx [hunk 20]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+
+export interface IProps {
+  text: string;
+}
+
+export const Description: FC<IProps> = ({ text }) => {
+  return (
```
Comment (garySiladi) [inline src/components/Thumbnail/Description.tsx]: remove `return` and simply let the const return the render as
```
export const Description: FC<IProps> = ({ text }) => (
  <p className="text-sm leading-normal font-lato font-normal mb-4">{text}</p>
  ....
```

File: src/components/Thumbnail/Button.tsx [hunk 21]
```diff
@@ -0,0 +1,13 @@
+import React, { FC } from "react";
+
+interface IProps {
+  text: string;
+}
+
+export const Button: FC<IProps> = ({ text }) => {
+  return (
```
Comment (garySiladi) [inline src/components/Thumbnail/Button.tsx]: remove `return` and simply let the const return the render as
```
export const Button: FC<IProps> = ({ text }) => (
  <button className="rounded-lg bg-transparent 
  ....
```

**Review by garySiladi — CHANGES_REQUESTED** [review]
some changes needed in renders

File: src/components/Thumbnail/Button.tsx [hunk 22]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+
+interface IProps {
+  text: string;
+}
+
+export const Button: FC<IProps> = ({ text }) => (
+  <button className="rounded-lg bg-transparent w-full h-10 font-black font-lato text-black hover:text-white hover:bg-black text-black font-black">
+    {text}
```
Comment (pggl) [inline src/components/Thumbnail/Button.tsx]: you have twice font-black and text-black, and why is here bg-transparent?

File: src/components/Thumbnail/Image.tsx [hunk 23]
```diff
@@ -0,0 +1,8 @@
+import React, { FC } from "react";
+
+interface IProps {
+  icon: string;
+}
+export const Image: FC<IProps> = ({ icon }) => (
+  <img className="mb-2" alt="therapy-Image" src={icon} />
```
Comment (pggl) [inline src/components/Thumbnail/Image.tsx]: why is alt attribute hardcoded?

File: src/components/Thumbnail/Name.tsx [hunk 24]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+
+interface IProps {
+  text: string;
+}
+
+export const Name: FC<IProps> = ({ text }) => (
+  <p className="font-Gilbert font-bold text-xl text-black leading-1.33">
```
Comment (pggl) [inline src/components/Thumbnail/Name.tsx]: do we have text-black?

File: src/components/Thumbnail/Thumbnail.tsx [hunk 25]
```diff
@@ -0,0 +1,17 @@
...
+import icon from "./homer.svg";
+
+export const Thumbnail = () => (
+  <div className="container ml-6">
+    <div className="p-2 w-1/5 h-1/3 m-2 shadow mr-2 mb-2 block">
+      <Image icon={icon} />
+      <Name text="MAMA CHAT" />
+      <Description text="Kreatívne a uvoľnujúce aktivity, aby ste si oddýchli, spracovali emócie […]" />
+      <Button text="Zistiť viac" />
+    </div>
```
Comment (pggl) [inline src/components/Thumbnail/Thumbnail.tsx]: we need props for icon, name, description and button

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/stories/index.tsx [hunk 26]
```diff
@@ -5,10 +5,32 @@ import { ErrorPopup } from "../components/Popup/Storybook/ErrorPopup";
 import { RegistrationPopup } from "../components/Popup/Storybook/RegistrationPopup";
 import { CancelReservationPopup } from "../components/Popup/Storybook/CancelReservationPopup";
 import { Header } from "../components/Header/Header";
+import icon from "../components/Thumbnail/homer.svg";
```
Comment (pggl) [inline src/stories/index.tsx]: move this file into different directory -> src/static/svg

File: src/components/Thumbnail/Thumbnail.tsx [hunk 27]
```diff
@@ -0,0 +1,29 @@
...
+  alt: string;
+}
+export const Thumbnail: FC<IProps> = ({
+  button,
+  description,
+  name,
+  icon,
+  alt
+}) => (
+  <div className="container ml-6">
```
Comment (pggl) [inline src/components/Thumbnail/Thumbnail.tsx]: why is here this div ?

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/Thumbnail/Button.tsx [hunk 28]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+
+interface IProps {
+  text: string;
```
Comment (pggl) [inline src/components/Thumbnail/Button.tsx]: use children instead for this

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/Thumbnail/Thumbnail.tsx [hunk 29]
```diff
@@ -0,0 +1,27 @@
...
+export const Thumbnail: FC<IProps> = ({
+  button,
+  description,
+  name,
+  icon,
+  alt
+}) => (
+  <div className="p-2 w-1/5 shadow block">
+    <Image icon={icon} alt={alt} classname="mb-2" />
+    <Name children={name} />
```
Comment (garySiladi) [inline src/components/Thumbnail/Thumbnail.tsx]: `children` are not a classic prop in this sense. we call `children` everything which is inside of the given component. check an article https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891
this should be written as 
```
<Name>{name}</Name>
```
Same with the following lines as in `Description` and `Button`

File: src/components/Thumbnail/Name.tsx [hunk 30]
```diff
@@ -0,0 +1,9 @@
+import React, { FC } from "react";
+
+interface IProps {
+  children: string;
```
Comment (garySiladi) [inline src/components/Thumbnail/Name.tsx]: this isn't a great practice, as children could be any renderable node
write:
```
import React, { FC, ReactNode } from "react";

interface IProps {
  children: ReactNode;
}
```
read for understanding https://medium.com/@martin_hotell/react-children-composition-patterns-with-typescript-56dfc8923c64

File: src/components/Thumbnail/Image.tsx [hunk 31]
```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+
+interface IProps {
+  icon: string;
+  alt: string;
+  classname?: string;
+}
+export const Image: FC<IProps> = ({ icon, alt, classname }) => (
```
Comment (garySiladi) [inline src/components/Thumbnail/Image.tsx:10]: set default `classname` value

File: src/components/Thumbnail/Button.tsx [hunk 32]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
```
Comment (garySiladi) [inline src/components/Thumbnail/Button.tsx:1]: the same `children` problem I've written in the other comments

**Review by garySiladi — CHANGES_REQUESTED** [review]

File: src/components/Thumbnail/Description.tsx [hunk 33]
```diff
@@ -0,0 +1,10 @@
+import React, { FC, ReactNode } from "react";
+
+export interface IProps {
+  children: ReactNode;
+  classname?: string;
+}
+
+export const Description: FC<IProps> = ({ children, classname = "" }) => (
+  <p className={`text-sm font-lato ${classname}`}>{children}</p>
```
Comment (garySiladi) [inline src/components/Thumbnail/Description.tsx]: Now that we've installed the `classnames` package, change this to: 
```
<p className={classnames('text-sm font-lato', classname)}>{children}</p>
```

File: src/components/Thumbnail/Image.tsx [hunk 34]
```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+
+interface IProps {
+  icon: string;
+  alt: string;
+  classname?: string;
+}
+export const Image: FC<IProps> = ({ icon, alt, classname = "" }) => (
+  <img alt={alt} src={icon} className={`w-full ${classname}`} />
```
Comment (garySiladi) [inline src/components/Thumbnail/Image.tsx]: the same

**Review by garySiladi — CHANGES_REQUESTED** [review]
pull master, and update according to comments

File: src/components/Thumbnail/Button.tsx [hunk 35]
```diff
@@ -0,0 +1,11 @@
+import React, { FC, ReactNode } from "react";
+
+interface IProps {
+  children: ReactNode;
```
Comment (garySiladi) [inline src/components/Thumbnail/Button.tsx]: remove `children` typing
the file can look like: 
```
import React, { FC } from "react";

export const Button: FC = ({ children }) => (
  <button className="rounded-lg w-full h-10 font-black font-lato hover:text-white hover:bg-black text-black">
    {children}
  </button>
);
```

File: src/components/Thumbnail/Description.tsx [hunk 36]
```diff
@@ -0,0 +1,11 @@
+import React, { FC, ReactNode } from "react";
+import classnames from "classnames";
+
+export interface IProps {
+  children: ReactNode;
```
Comment (garySiladi) [inline src/components/Thumbnail/Description.tsx]: remove children type

File: src/components/Thumbnail/Description.tsx [hunk 37]
```diff
@@ -0,0 +1,11 @@
+import React, { FC, ReactNode } from "react";
```
Comment (garySiladi) [inline src/components/Thumbnail/Description.tsx]: remove ReactNode as not needed anymore

File: src/components/Thumbnail/Image.tsx [hunk 38]
```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+import classnames from "classnames";
+interface IProps {
+  icon: string;
+  alt: string;
+  classname?: string;
+}
```
Comment (garySiladi) [inline src/components/Thumbnail/Image.tsx:8]: add line here

File: src/components/Thumbnail/Name.tsx [hunk 39]
```diff
@@ -0,0 +1,9 @@
+import React, { FC, ReactNode } from "react";
```
Comment (garySiladi) [inline src/components/Thumbnail/Name.tsx]: same here with the removal of typing
```
import React, { FC } from "react";

export const Name: FC = ({ children }) => (
  <p className="font-gilbert text-xl text-black">{children}</p>
);
```

**Review by garySiladi — CHANGES_REQUESTED** [review]

File: src/components/Thumbnail/Description.tsx [hunk 40]
```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+import classnames from "classnames";
+
+export interface IProps {
+  classname?: string;
+}
+
+export const Description: FC<IProps> = ({ children, classname = "" }) => (
```
Comment (pggl) [inline src/components/Thumbnail/Description.tsx]: we don't need now default value for classname because classnames library fix this

File: src/components/Thumbnail/Description.tsx [hunk 41]
```diff
@@ -0,0 +1,10 @@
...
+  <p className={classnames("text-sm font-lato", classname)}>{children}</p>
```
Comment (pggl) [inline src/components/Thumbnail/Description.tsx]: we need handle long texts, use text-overflow: ellipsis;
https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

File: src/components/Thumbnail/Image.tsx [hunk 42]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+import classnames from "classnames";
+
+interface IProps {
+  icon: string;
+  alt: string;
+  classname?: string;
+}
+export const Image: FC<IProps> = ({ icon, alt, classname = "" }) => (
```
Comment (pggl) [inline src/components/Thumbnail/Image.tsx]: we don't need now default value for classname because classnames library fix this

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/Thumbnail/Description.tsx (hunk 41 above)
Comment (slavomircesla) [reply src/components/Thumbnail/Description.tsx]: ellipsis is not supported on tailwind, had to use "whitespace-normal overflow-hidden"

File: src/components/Thumbnail/Thumbnail.tsx [hunk 43]
```diff
@@ -0,0 +1,27 @@
...
+  alt: string;
+}
+export const Thumbnail: FC<IProps> = ({
+  button,
+  description,
+  name,
+  icon,
+  alt
+}) => (
+  <div className="p-2 w-1/5 shadow block">
```
Comment (pggl) [inline src/components/Thumbnail/Thumbnail.tsx]: remove w-1/5, block

File: src/components/Thumbnail/Image.tsx [hunk 44]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+import classnames from "classnames";
```
Comment (pggl) [inline src/components/Thumbnail/Image.tsx:2]: remove this

File: src/components/Thumbnail/Description.tsx [hunk 45]
```diff
@@ -0,0 +1,12 @@
+import React, { FC } from "react";
+import classnames from "classnames";
```
Comment (pggl) [inline src/components/Thumbnail/Description.tsx:2]: remove this

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/Thumbnail/Description.tsx (hunk 41 above)
Comment (garySiladi) [reply src/components/Thumbnail/Description.tsx]: tailwind has a special `truncate` class for this, find it here: https://tailwindcss.com/docs/word-break/#app

File: src/components/Thumbnail/Description.tsx [hunk 46]
```diff
@@ -0,0 +1,19 @@
...
+
+interface IProps {
+  classname?: string;
+}
+export const Description: FC<IProps> = ({ children, classname }) => (
+  <div>
```
Comment (pggl) [inline src/components/Thumbnail/Description.tsx]: remove div

File: src/components/Thumbnail/Name.tsx [hunk 47]
```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+
+export const Name: FC = ({ children }) => (
+  <span
+    className="font-gilbert flex text-xl text-black w-full"
+    style={{ height: "28px" }}
```
Comment (pggl) [inline src/components/Thumbnail/Name.tsx]: remove

File: src/components/Thumbnail/Name.tsx [hunk 48]
```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+
+export const Name: FC = ({ children }) => (
+  <span
```
Comment (pggl) [inline src/components/Thumbnail/Name.tsx]: use <p>

File: src/components/Thumbnail/Thumbnail.tsx [hunk 49]
```diff
@@ -0,0 +1,20 @@
...
+  name: string;
+  icon: string;
+  alt: string;
+}
+export const Thumbnail: FC<IProps> = ({ description, name, icon, alt }) => (
+  <div className="p-2 w-full shadow" style={{ height: "304px" }}>
+    <Image icon={icon} alt={alt} classname="mb-2" />
+    <Name>{name}</Name>
+    <Description classname="mb-4">{description}</Description>
+    <Button>Zistiť viac</Button>
```
Comment (pggl) [inline src/components/Thumbnail/Thumbnail.tsx:18]: use internal component

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/Thumbnail/Thumbnail.tsx (hunk 49 above)
Comment (slavomircesla) [reply src/components/Thumbnail/Thumbnail.tsx:18]: Need create second internal Button component

**Review by garySiladi — DISMISSED** [review]
from my side ok, if that button question was sorted out with @pggl

File: src/components/Thumbnail/Description.tsx [hunk 50]
```diff
@@ -0,0 +1,17 @@
...
+import classnames from "classnames";
+
+interface IProps {
+  classname?: string;
+}
+export const Description: FC<IProps> = ({ children, classname }) => (
+  <p
+    style={{
+      height: "36px",
+      lineHeight: "18px"
```
Comment (pggl) [inline src/components/Thumbnail/Description.tsx]: use tailwind

File: src/components/Thumbnail/Description.tsx [hunk 51]
```diff
@@ -0,0 +1,17 @@
...
+    }}
+    className={classnames("text-sm font-lato w-auto", classname)}
```
Comment (pggl) [inline src/components/Thumbnail/Description.tsx]: I think w-auto is useless

File: src/components/Thumbnail/Button.tsx [hunk 52]
```diff
@@ -0,0 +1,7 @@
+import React, { FC } from "react";
+
+export const Button: FC = ({ children }) => (
+  <button className="rounded-lg w-full mb-2 h-10 font-black font-lato hover:text-white hover:bg-black text-black">
```
Comment (pggl) [inline src/components/Thumbnail/Button.tsx]: mb-2 is useless

**Review by pggl — CHANGES_REQUESTED** [review]

**Review by pggl — APPROVED** [review]
